pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>org.exist-db</groupId>
     <artifactId>exist-apps-parent</artifactId>
-    <version>1.9-SNAPSHOT</version>
+    <version>1.10-SNAPSHOT</version>
```

Comment:
This should never be changed manually. Instead we let the release plugin handle it.

Reply:
it was release plugin, that handles that, but a release artifact still needs to be pulled from master